src/extension/postgres/ltree.rs

```diff
@@ -0,0 +1,34 @@
+use std::convert::TryFrom;
+use std::fmt;
+
+use crate::PgLTree;
+
+impl fmt::Display for PgLTree {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}", self.0)
+    }
+}
+
+impl TryFrom<&str> for PgLTree {
+    type Error = String;
+
+    fn try_from(field: &str) -> Result<Self, Self::Error> {
+        Ok(PgLTree(field.to_owned()))
+    }
+}
+
+impl TryFrom<String> for PgLTree {
+    type Error = String;
+
+    fn try_from(value: String) -> Result<Self, Self::Error> {
+        Ok(PgLTree(value))
+    }
+}
+
+impl TryFrom<&String> for PgLTree {
+    type Error = String;
+
+    fn try_from(value: &String) -> Result<Self, Self::Error> {
+        Ok(PgLTree(value.to_owned()))
+    }
+}
```

Comment:
It's `TryFrom` because paths from a LTree are stored in string which format splits every node with a dot.
E.x:

```
electronics.video-games.playstation
```

I was thinking about validating this string so we ensure the path has the correct format in case the value was manually changed. But perhaps by deferring database operations we don't have to do manual validation, wdyt?

Reply:
Addressed here: [33436e4](https://github.com/SeaQL/sea-query/pull/604/commits/33436e4fc97e60fa2a69f30a0287550c375f3a81)